### ISSUE TITLE
Webpack Error

### ISSUE THREAD

**Issue by zerolfc** [burt202/lite-uploader#30]
Im using Laravel Elixir Webpack to compile all the javascript libraries and scripts.

They are all fine except, i keep getting the error. 

Any advise on how to fix it?

`php
app.js?8b67:146 Uncaught TypeError: $(...).liteUploader is not a function(…)`

![webpack](https://cloud.githubusercontent.com/assets/5167436/20772379/1f71b4d0-b745-11e6-9ae1-7ff3a75e454e.PNG)

**Comment by burt202** [comment]
Hi, I have no experience with `Laravel Elixir Webpack` so cant help out here directly but I can confirm I personally have used it with the standard `webpack` package with no problem. Can you paste/gist a real simple example with the error so I could take a look and hopefully lend a hand? Cheers

**Comment by zerolfc** [comment]
hello, 
thanks for the reply, not sure the following code will help.
here is the code: 

```
window.$ = window.jQuery = require('jquery');
require('bootstrap-sass');
require('lite-uploader/src/liteuploader.js);

$(function(){
  $('.file-uploader').liteUploader();
  $(".file-uploader").change(function () {
    $(this).data("liteUploader").startUpload();
  });
});
```

in gulpfile.js
```
const elixir = require('laravel-elixir');
elixir((mix) => {
    mix.sass('app.scss')
       .webpack('app.js');
});
```

**Comment by burt202** [comment]
Ive pushed a fix for this, and I'll aim to publish it later.
If it helps the main two parts of my test are below. See my last 2 commits for the fixes

Hope this helps

app.js
```
var $ = require("jquery")
require("lite-uploader")

$(".fileUpload").liteUploader({
  script: "http://localhost:8000/test/test.php"
})

$(".fileUpload").change(function () {
  $(this).data("liteUploader").startUpload();
});

```

webpack.config.js
```
var webpack = require('webpack')

module.exports = {
  entry: [
    "./app.js"
  ],
  output: {
    filename: "bundle.js",
    path: __dirname
  },
  plugins: [
    new webpack.ProvidePlugin({
      $: "jquery",
      "window.$": "jquery"
    })
  ]
}
```

**Comment by burt202** [comment]
Published v5.0.2. Let me know if this fixes it for you. Cheers

**Comment by zerolfc** [comment]
hi, you fixed it. thank you :)
